src/modclub/service/poh/poh.mo

```diff
@@ -674,7 +687,8 @@ module PohModule {
                 return #alreadySubmitted;
               } else if (attempts.get(attempts.size() -1).status == #verified) {
                 return #alreadyApproved;
-              } else if (attempts.get(attempts.size() -1).status == #rejected) {
+              } else if (attempts.get(attempts.size() -1).status == #rejected or 
+              attempts.get(attempts.size() -1).status == #rejectedDuplicate) {
```

Comment:
attempts is a buffer so expression like 
`attempts.get(attempts.size() -1)`
will crush the app if `buffer.size() // => 0`